projects/aca-folder-rules/src/lib/rule-set-picker/rule-set-picker.smart-component.ts

```diff
@@ -59,6 +60,8 @@ export class RuleSetPickerSmartComponent {
     map(([rulesLoading, folderLoading]) => rulesLoading || folderLoading)
   );
 
+  onDestroy$: Subject<boolean> = new Subject<boolean>();
```

Comment:
can be void instead of boolean

Reply:
Removed boolean from subject. New declaration is

`onDestroy$ = new Subject();`

Reply:
You have still boolean at the left side (you don't need specify type at the left side when you have constructor at right side):   onDestroy$: Subject<boolean> = new Subject();

Also please see my comment here: https://github.com/Alfresco/alfresco-content-app/pull/3227#discussion_r1206318128

Reply:
Updated declaration to be 
```
onDestroy$ = new Subject<void>();
```
Additionally, removed the type from the left side